synapse/config/sso.py

```diff
@@ -74,6 +74,8 @@ def read_config(self, config, **kwargs):
 
         self.sso_client_whitelist = sso_config.get("client_whitelist") or []
 
+        self.sso_update_display_name = sso_config.get("update_display_name") or False
+
```

Comment:
Additionally, I wonder if it's likely that we'll ever add any more auto-updating information, such as user avatars. It'd be ideal to have both under the same option. Perhaps `update_profile_information` may be more future-proof?

Reply:
There's also e-mails...

Reply:
Indeed, which are already implemented. Seems best to generalise the option name then, but to document that only display names are transferred for now.

How about `sync_sso_profile_information`?